File: backend/src/main/java/etf/bg/diplomski/lesson/LessonRequest.java
package etf.bg.diplomski.lesson;

import etf.bg.diplomski.student.StudentDTO;
import etf.bg.diplomski.subject.SubjectNameDTO;

import java.sql.Time;
import java.time.LocalDate;
import java.util.List;

public record LessonRequest(
        Long lessonId,
        List<StudentDTO> student,
        List<Double> studentsScore,
        SubjectNameDTO subject,
        LocalDate date,
        Time time,
        String description) {
}

File: vue_front/src/models/lessonRequest.js
import { Time } from "@angular/common";
import { SubjectDTO } from "./SubjectDTO";
import { Student } from "./Student";

class LessonRequest{
    date(){
        return {
        lessonId:number
        student:[],
        studentsScore:number,
        subject:SubjectDTO,
        date:Date,
        time:Time,
        description:""}
    }
}

export default new LessonRequest